jira/client.py

```diff
@@ -1591,14 +1591,18 @@ def assign_issue(self, issue, assignee):
         return True
 
     @translate_resource_args
-    def comments(self, issue):
+    def comments(self, issue, expand=None):
         """Get a list of comment Resources.
 
         :param issue: the issue to get comments from
+        :param expand: extra information to fetch inside each comment
```

Comment:
are these fields? maybe add that too if these are field names 
maybe provide an example?

Reply:
Just updated the comment's indicating what can be specified, e.g. renderedBody or properties.